copro/selection.py

```diff
@@ -131,9 +135,9 @@ def climate_zoning(gdf, extent_gdf, config, root_dir):
         # get only those entries with retrieved codes
         KG_gdf = KG_gdf.loc[KG_gdf['GRIDCODE'].isin(code_nrs)]
         
-        # make sure EPSG:4236 is used
-        if KG_gdf.crs != 'EPSG:4326':
-            KG_gdf = KG_gdf.to_crs('EPSG:4326')
+        # make sure EPSG:4236 is used --> now changed into WGS84
+        if KG_gdf.crs != 'WGS84':
```

Comment:
Why using `WGS84` instead of `EPSG4326`?

Reply:
Omdat de CRS van de migratiedata in WGS84 is, en er als Shapefile in gaat

Reply:
Ok prima - kan je mergen!